pkg/processor/test/suite/suite.go

```diff
@@ -60,6 +61,19 @@ type TestSuite struct {
 	CleanupTemp  bool
 }
 
+// StressRequest holds information for blastHTTP function
```

Comment:
Make sure that things have one name in a given context. In this case BlastHTTP should receive `BlastRequest`, not `StressRequest`. In addition, I think it's more consistent to call this `BlastConfiguration`.

In addition, use the pattern of `NewX` to initialize stuff. In this case a package levle function `NewBlastConfiguration` rather than `GetDefault...` which is part of `TestSuite`.